libs/sdk-core/src/breez_services.rs

```diff
@@ -1400,6 +1421,8 @@ impl Receiver for PaymentReceiver {
                         )?,
                         incoming_amount_msat: amount_msats as i64,
                         outgoing_amount_msat: (destination_invoice_amount_sats * 1000) as i64,
+                        tag: json!({ "apiKeyHash": api_key_hash }).to_string(),
+                        opening_fee_params: channel_opening_fee_params.clone().map(Into::into),
```

Comment:
Let's add a little check inside this `if open_channel_needed` statement that `channel_opening_fee_params` is never `None` here to avoid any future bugs where we may accidentally pass None.

Reply:
Done: be5f6ab922e1408d10c2ccf19c29183afd22b35b